.buildkite/pipelines/pull_request/security_solution/defend_workflows.yml

```diff
@@ -7,10 +7,10 @@ steps:
       - build
       - quick_checks
     timeout_in_minutes: 60
-    parallelism: 16
```

Comment:
Can we also check for updates on:
- .buildkite/pipelines/on_merge.yml
- .buildkite/pipelines/es_serverless/verify_es_serverless_image.yml